cordova-lib/src/plugman/plugman.js

```diff
@@ -89,9 +90,12 @@ plugman.commands =  {
         plugman.owner(cli_opts.argv.remain);
     },
     'install'  : function(cli_opts) {
+        resolvePathsInArgs(cli_opts);
```

Comment:
CLI code doesn't pass through here either I don't think. Within `install.js` is where you should do it I think.